src/app/components/form/DualCurrencyField/index.tsx

```diff
@@ -61,6 +61,15 @@ export default function DualCurrencyField({
     />
   );
 
+  // run effect on input mount to ignoreScroll event
+  useEffect(() => {
+    if (inputEl.current == null) return;
+    const ignoreScroll = (evt: globalThis.WheelEvent) => {
+      evt.preventDefault();
+    };
+    inputEl.current.addEventListener("wheel", ignoreScroll);
```

Comment:
Could the `if` above be avoided by doing this here?:
```ts
inputEl.current && inputEl.current.addEventListener("wheel", ignoreScroll);
```

Reply:
Yes @escapedcat It could be. 
Should I do that?

Reply:
Yes, that would be nice, thanks!